vscode/package.json

```diff
@@ -270,6 +275,11 @@
         "title": "Help",
         "category": "Q#"
       },
+      {
+        "command": "qsharp-vscode.showCircuit",
+        "title": "Show circuit",
+        "category": "Q#"
```

Comment:
I think this shows up by default in the Command Palette unless you add a condition (in this case, should be conditions on the config value to enable it). See <https://code.visualstudio.com/api/extension-guides/command#controlling-when-a-command-shows-up-in-the-command-palette>